### PR TITLE
Add youtube clone

### PR DESCRIPTION
**Pull request by BasitKorede**
#### Reason for doing this project:
  This project is an assignment assigned to improve ones skill on the knowledge of  HTML and CSS.
The PR does the following:
- Follows the guidelines on [Odin project assignment page](https://www.theodinproject.com/courses/html-and-css/lessons/embedding-images-and-video) to  embed video and images.

#### things learnt  from this project
- Learnt how to embed a video into my website
- Learnt how to embed a images into my website
- Learnt how to effectively use the developer tools
- Now know how to effectively use Github
- [Live Demo Link](https://deploy-preview-1--basit-youtube-clone.netlify.app/)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+
+@media screen and (max-width: 993px) {
+  .main {
+    width: 100%;
+    display: block;
+  }
+
+  .main-video {
+    width: 100%;
+  }
```
Comment (adejam) [inline css/style.css:255]: For future projects, I would recommend using CSS imports for media queries, see [here](https://developer.mozilla.org/en/docs/Web/CSS/@import) and [here](https://www.w3schools.com/cssref/pr_import_rule.asp) for more details.

CSS imports help with code splitting, among other benefits, it makes your code more organized.

**Review by adejam — CHANGES_REQUESTED** [review]
Good job with the project!

Kindly check the comment below and make changes accordingly.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,33 @@
...
+> This project is assigned by a plattform (THE ODIN PROJECT) under the (course HTML and CSS).
 
-this project is an assignment,assigned by a platform (THE ODIN PROJECT)
-under the HTML and CSS course. 
+![screenshot](./clone.png)
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+This project is an assignment on enbbeding videos and images to your website .
```
[the remark this answers is not in the document]
Comment (BasitKorede) [reply README.md:7]: ok thanks

File: README.md [hunk 3]
```diff
@@ -1,37 +1,33 @@
...
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://basit-youtube-clone.netlify.app/)
 
 ### Development (Running locally)
 
 - Clone the project
 
 ```bash
-git clone https://github.com/user/html-css-template.git
+git clone https://github.com/BasitKorede/youtube-clone
```
[the remark this answers is not in the document]
Comment (BasitKorede) [reply README.md:23]: on it

File: README.md [hunk 4]
```diff
@@ -89,9 +83,8 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration, resources/assets used
-- etc
+- [@adejam](https://github.com/adejam)
+- [@bolah2009](https://github.com/bolah2009)
```
[the remark this answers is not in the document]
Comment (BasitKorede) [reply README.md:87]: ok thanks

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+
+.navi {
+  display: flex;
+  margin-bottom: 30px;
+  justify-content: space-between;
+}
+
+.navi ul {
+  display: flex;
+}
```
[the remark this answers is not in the document]
Comment (BasitKorede) [reply css/style.css:13]: thanks for the recommendation

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+.input {
+  width: 80%;
+  height: 30px;
+  margin-right: 0;
+  margin-left: 20%;
+  margin-bottom: 20px;
+  margin-top: 20px;
+  border: 1px solid #808080;
+  border-right: none;
+}
```
[the remark this answers is not in the document]
Comment (BasitKorede) [reply css/style.css:81]: ok

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+
+.search-botton {
+  width: 60px;
+  height: 30px;
+  margin-left: 0%;
+  margin-right: 15%;
+  margin-top: 20px;
+  border-left: 1px solid #808080;
+  border: 1px solid #808080;
+}
```
[the remark this answers is not in the document]
Comment (BasitKorede) [reply css/style.css:100]: ok

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+
+.updates {
+  display: flex;
+  justify-content: space-between;
+}
+
+.cont {
+  display: flex;
+  flex-grow: 1;
+}
```
[the remark this answers is not in the document]
Comment (BasitKorede) [reply css/style.css:231]: sure,thanks

File: css/style.css (hunk 1 above)
Comment (BasitKorede) [reply css/style.css:255]: thanks for the recommendation.i will surely do that

**Comment by BasitKorede** [comment]
Thanks for the review. I will amend my mistakes

File: README.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # YOUTUBE-CLONE
 
-> This project is assigned by a plattform (THE ODIN PROJECT) under the (course HTML and CSS).
+> This project followes the guilde line of [THE ODIN PROJECT](https://www.theodinproject.com/courses/html-and-css/lessons/grid) under the (course HTML and CSS) to clone a youtube page [YouTube](https://www.youtube.com/watch?v=Z_sVBF68N9o).
```
Comment (adejam) [inline README.md:3]: You should pay attention to incorrect spellings and/or typo errors in your documentation. This is in reference to "*guilde line"* which ought to be *"guideline"*.

**Review by adejam — APPROVED** [review]
LGTM 

Awesome Job!!!

Please make the following changes before merging.

**Pull request merged**